README.md

```diff
@@ -64,7 +71,7 @@ Everyone:
         $ mvn clean install
 
 ####Building an ear
-All services profiles are active by default, so to build an ear containing all services, just execute:
+All services profiles are active by default, so to build an ear containing all services, except for Direct, X12 just execute:
```

Comment:
Suggestion, possibly change "an ear" to "a GlassFish ear" (since that's the only one built by default).